samples/LargeResponseApp/LargeResponseApp.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web" ToolsVersion="15.0">
+  <Import Project="..\..\build\common.props" />
```

Comment:
Copy in the stuff it needs. In theory, we want users to be able to copy the sample and run without needing Internal.AspNetCore.Sdk.